### PR TITLE
Docs: sync and refresh ask stream operators

### PR DESCRIPTION
**Pull request by ennru**
Review of the texts and examples for the classic and the typed variants of the `ask` stream operator.

Move the classic operator into the [Actor Interop](https://doc.akka.io/docs/akka/snapshot/stream/operators/#actor-interop-operators) section.

Move the sample source code into `docs.(javadsl|scaladsl)` to show all imports.

References #25468

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md [hunk 1]
```diff
@@ -14,25 +14,48 @@ This operator is included in:
...
 
-@@signature [ActorFlow.scala](/akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorFlow.scala) { #ask }
-
-@@@
+@apidoc[ActorFlow.ask](ActorFlow$) { scala="#ask%5BI,Q,A](ref:akka.actor.typed.ActorRef%5BQ])(makeMessage:(I,akka.actor.typed.ActorRef%5BA])=%3EQ)(implicittimeout:akka.util.Timeout):akka.stream.scaladsl.Flow%5BI,A,akka.NotUsed]" java="#ask(akka.actor.typed.ActorRef,java.time.Duration,java.util.function.BiFunction)" }
 
 ## Description
 
-Emit the contents of a file, as `ByteString`s, materializes into a @scala[`Future`] @java[`CompletionStage`] which will be completed with
-a `IOResult` upon reaching the end of the file or if there is a failure.
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md]: No idea how this description ended up in the `ask` docs.

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # ActorFlow.ask
 
-Use the `AskPattern` to send each element as an `ask` to the target actor, and expect a reply back that will be sent further downstream.
+Use the "Ask Pattern" to send each stream element as an `ask` to the target actor (of the new actors API), and expect a reply back that will be emitted downstream.
```
Comment (ignasi35) [inline akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md:3]:
```suggestion
Use the "Ask Pattern" to send each stream element as an `ask` to the target actor (of the Typed Actors API), and expect a reply back that will be emitted downstream.
```

**Review by ignasi35 — APPROVED** [review]

**Review by ignasi35 — CHANGES_REQUESTED** [review]
(sorry)

Left a small improvement as a suggestion. Otherwise LGTM.

File: akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md (hunk 2 above)
Comment (ennru) [reply akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md:3]: We try to use "new actors API" consistently.

**Review by ignasi35 — APPROVED** [review]

**Pull request merged**